tests/jobs/test_scheduler_job.py

```diff
@@ -75,8 +76,7 @@
 )
 PERF_DAGS_FOLDER = os.path.join(ROOT_FOLDER, "tests", "test_utils", "perf", "dags")
 ELASTIC_DAG_FILE = os.path.join(PERF_DAGS_FOLDER, "elastic_dag.py")
-
-TEST_DAG_FOLDER = os.environ['AIRFLOW__CORE__DAGS_FOLDER']
+TEST_DAG_FOLDER = '/airflow/github/dags'
```

Comment:
Why is this change?

Reply:
Good catch. It's just some leftover from my tests.
Reverted.